earth_enterprise/src/SConstruct

```diff
@@ -176,7 +176,10 @@ el_ver = ''
 if os.path.isfile('/etc/redhat-release') and not os.path.isfile('/etc/os-release'):
   el_ver = 'el6'
 else:
-  el_ver = 'el7'
+  if 'el8' in os.uname()[2]:
+    el_ver = 'el8'
+  else:
+    el_ver = 'el7'
```

Comment:
might be worthwhile to make `png_ver` an env variable so we can account for Ubuntu later on